exporter/stackdriverexporter/config.go

```diff
@@ -27,5 +27,20 @@ type Config struct {
 	NumOfWorkers                  int                      `mapstructure:"number_of_workers"`
 	SkipCreateMetricDescriptor    bool                     `mapstructure:"skip_create_metric_descriptor"`
 	// Only has effect if Endpoint is not ""
-	UseInsecure bool `mapstructure:"use_insecure"`
+	UseInsecure      bool              `mapstructure:"use_insecure"`
+	ResourceMappings []ResourceMapping `mapstructure:"resource_mappings"`
+}
+
+// ResourceMapping defines mapping of resources from source (OpenCensus) to target (Stackdriver).
+type ResourceMapping struct {
+	SourceType string `mapstructure:"source_type"`
+	TargetType string `mapstructure:"target_type"`
+
+	LabelMappings []LabelMapping `mapstructure:"label_mappings"`
+}
+
+type LabelMapping struct {
+	SourceKey string `mapstructure:"source_key"`
+	TargetKey string `mapstructure:"target_key"`
+	Optional  bool   `mapstructure:"optional"`
```

Comment:
Can you document the semantics of this parameter? When would one set it to true?

Reply:
It just signals that we can proceed with transformation when a particular label with a given `source_key` is missing.
I will add a comment.

Reply:
Done.